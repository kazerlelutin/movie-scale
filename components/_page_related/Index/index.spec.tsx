import { render} from "@testing-library/react";
import Home from "./Index";
import "@testing-library/jest-dom";
import { SessionProvider } from "next-auth/react";

describe("Home", () => {
  it("renders homepage unchanged", () => {
    const { container } = render(
      <SessionProvider session={null}>
        <Home />
      </SessionProvider>
    );
    expect(container).toMatchSnapshot();
  });
});
