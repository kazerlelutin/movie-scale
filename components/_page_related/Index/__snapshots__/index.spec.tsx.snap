// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders homepage unchanged 1`] = `
<div>
  <div
    class="container"
  >
    <header
      class="header"
    >
      <span
        style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
      >
        <span
          style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
        >
          <img
            alt=""
            aria-hidden="true"
            src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2750%27%20height=%2729%27/%3e"
            style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
          />
        </span>
        <img
          alt="logo"
          data-nimg="intrinsic"
          decoding="async"
          src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
          style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
        />
        <noscript />
      </span>
      <div
        class="container"
      >
        <button
          class="connect"
        >
          Se connecter
        </button>
        <button>
          S'enregistrer
        </button>
      </div>
    </header>
    <main>
      <div
        class="container"
      >
        <header
          class="header"
        >
          <button>
            Créer une échelle
          </button>
          <div
            class="description"
          >
            Un échelle permet de classer des films par ordre de préférence
          </div>
          <input
            class="header"
            placeholder="Rechercher une échelle"
            type="text"
          />
        </header>
        <section
          class="scales"
        >
          Les échelles populaires
        </section>
      </div>
    </main>
  </div>
</div>
`;
